CodeStarterCamp_Week1/main.swift

```diff
@@ -8,5 +8,49 @@
 
 import Foundation
 
-print("Hello, World!")
+// Step 2
+
+func icecream1(ice_row: Int, ice_col: Int, bar_row: Int) {
+    for ice_r in 1...ice_row {
+        for ice_c in 1...ice_col {
+            print("*", terminator: "")}
```

Comment:
스위프트에선 일반적으로 코드블럭의 끝은 개행하는 편입니다! 특별한 경우를 제외하구요!
```suggestion
            print("*", terminator: "")
        }
```